### PR TITLE
Adding a file for waltham ma

### LINKED ISSUE #23: svatasoiumms Recommendations
add me

### PR DESCRIPTION
**Pull request by svatasoiumms**
This Pull Request resolves #23 by doing nothing

...
please comment on if you like @ciaran036

### CODE REVIEW

File: svatasoiumms-waltham.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test file
```
Comment (svatasoiumms) [inline svatasoiumms-waltham.md]: adding comment

Comment (svatasoiumms) [inline svatasoiumms-waltham.md]: reply to comment

Comment (nmcallister01) [inline svatasoiumms-waltham.md]: Is there any soccer stadiums nearby?

Comment (svatasoiumms) [inline svatasoiumms-waltham.md]: yes, foxboro MA

**Pull request merged**